### DISCUSSION TITLE
New Event: Game Framerate

### DISCUSSION THREAD

**Discussion by ghost** [CapsAdmin/pac3#1228] [Ideas]
Thought this might be a good idea to have for pacs with a ton of parts.

Basically this event would only trigger once your framerate reaches a set value kind of like the speed event where you can set it to be below, equal or above.

I think this would be useful to allow people to somewhat optimise laggy pacs or just disable specific parts if the framerate dips too low.

For example if I were to have  jiggles, proxies, lights, animated textures, etc I could disable them automatically if my framerate were to get too low and then reenable them once it has increased to a specific point.

![Frame Rate Event](https://user-images.githubusercontent.com/20357923/189916844-8895edeb-9d36-4033-8a45-22a28e7a6732.png)

**Comment by thegrb93** [comment]
Disabling and reenabling won't work, because as soon as its disabled, your framerate goes up, which reenables it, which lowers framerate, which disables it, etc.

**Reply by ghost** [reply]
> Disabling and reenabling won't work, because as soon as its disabled, your framerate goes up, which reenables it, which lowers framerate, which disables it, etc.

You could use the event alongside a command part, so you can have it so it will disable when at 30 frames but then only reenable at 60.
